ckoga/SWAPI-Study-Sesh
# Styling for CharacterCards and scroll text

### What's this PR do?
This PR finishes styling for the text scroll when CharacterCard component is rendered.  It also fixes styling for list items and line height on the CharacterCards.
### Where should the reviewer start?
There are 5 files with changes the you'll see important changes in the DisplayContainer rendering of CharacterCards I had to add another div to hold the scroll and prevent the cards from being placed next to the text because of display flex.  Other things to look at are the individual scss files where you will see rule additions for various styling.
### How should this be manually tested?
Pull this branch down and sign in pick a movie and you will be able to see the changes made to scroll text and the CharacterCards.
